### PR TITLE
Init commit with redis

### PR DESCRIPTION
**Pull request by ArcherUz**

### CODE REVIEW

**Comment by LeonYanc** [comment]
The whole .idea/ should be put into .gitignore and shouldn't be commited.

**Comment by LeonYanc** [comment]
Another comment: pls add screen shots as test evidence

**Comment by ArcherUz** [comment]
.idea is removed with commands added

**Comment by ArcherUz** [comment]
I removed .idea and added comments in my commits.
You can check those comments at https://github.com/ArcherUz/ZipURL-Services/commits/initCommitWithRedis/
I am not sure if they can be viewed in merged repo

**Comment by LeonYanc** [comment]
I saw you are using maven, which is deprecated and should replace it with gradle. Please replace maven with gradle. You can use Spring boot initializer to create a empty project and move your logic into it.
image

**Comment by ArcherUz** [comment]
I have replaced with gradle in this commit.
Once this is merged, I will PR from a new branch which implements JWT token.

File: urlsconvert/mvnw
Comment (LeonYanc) [inline urlsconvert/mvnw:1]: Remove this file

File: urlsconvert/mvnw.cmd
Comment (LeonYanc) [inline urlsconvert/mvnw.cmd:1]: Remove this file

File: urlsconvert/.mvn/wrapper/maven-wrapper.jar
Comment (LeonYanc) [inline urlsconvert/.mvn/wrapper/maven-wrapper.jar:1]: Do we still need this file?

File: urlsconvert/.mvn/wrapper/maven-wrapper.properties
Comment (LeonYanc) [inline urlsconvert/.mvn/wrapper/maven-wrapper.properties:1]: Do we still need this file?

**Comment by LeonYanc** [comment]
Add some more comments on cleaning up the remaining maven files.

**Review by LeonQC — APPROVED** [review]
LGTM

**Review by LeonYanc — APPROVED** [review]

**Pull request merged**